doc/contribute/committer_expectations.rst

```diff
@@ -0,0 +1,165 @@
+  feedback.
+
+- When breaking a PR into multiple commits, each commit must build cleanly. This
+  is not enforced by the CI system, so it is the PR author’s responsibility to
+  verify.
+
+- When major new functionality is added, tests for the new functionality MUST be
+  added to the automated test suite. All new APIs MUST be documented and tested
+  and tests MUST cover at least 80% of the added functionality using the code
+  coverage tool and reporting provided by the project..
+
+- Incompatible changes to APIs must also update the release notes for the
+  next release detailing the change.  APIs marked as experimented are excluded
+  from this requirement. [#api-updates]_
+
+- PRs must also satisfy all :ref:`merge_criteria` before a member of the release
+  engineering team will merge the PR into the zephyr tree.
+
+Maintainers may request you break up a PR into smaller PRs and may request you
+create an :ref:`RFC proposal <rfcs>`.
+
+Workflow Suggestions That Help Reviewers
+========================================
+
+- Don’t resolve comments until the question has been answered or the request
+  completed in an update to the PR.
+
+- Try to minimize rebases in the middle of a review. If a rebase is required,
+  push this as a separate update with no functional changes to the PR. See the
+  note below.
+
+- Instead of amending commits, append fixup commits to the PR to address
+  reviewer comments. This provides reviewers a clear history of the changes made
+  to the PR, along with the rationale for changes. Prior to the final approval
+  and merge of the PR, you must perform a squash merge to cleanup the commit
+  history. (TBD - This needs a PoC to demonstrate this workflow).
+
+Reviewer Expectations
+#####################
+
+- Be respectful when commenting on PRs. Refer to the Zephyr `Code of Conduct`_
+  for more details.
+
+- Review PRs within a reasonable time. (TBD - define soft guidelines for
+  “reasonable”)
+
+- Try to provide feedback on the entire PR in one shot. This provides the author
+  an opportunity to address all comments in the next PR update.
+
+- Partial reviews which focus on a domain (e.g. Devicetree) are permitted, but
+  the reviewer must add a comment indicating what portion of the PR they
+  reviewed. (TBD - ideally we could designate multiple assignees on a PR to
+  cover cross-domain PRs).
+
+- Avoid increasing scope of the PR by requesting new features, especially when
+  there is a corresponding :ref:`RFC <rfcs>` associated with the PR. Instead add
+  your suggestions as a comment to the :ref:`RFC <rfcs>`. This also encourages
+  more collaboration as it is easier for multiple contributors to work on a
+  feature once the minimum implementation has merged.
+
+- Reviewers should remove themselves from the PR if they are unable to complete
+  a review either due to lack of domain knowledge or cannot complete a review in
+  a timely fashion.
```

Comment:
And enter into a fight that you can't win against zephyr-bot? I think this is impractical.

Reply:
Zephyrbot assigns too many reviewers in my opinion.  I was looking for a mechanism for the reviewers to signal to the author (and other reviewers) that they do not plan on looking at a PR.  Removing this language for now.

Only adding a comment stating you don't plan to review doesn't really help move a PR forward.  Github will also continue to spam the uninterested reviewer on any comments or force pushes.